unicodetools/src/main/java/org/unicode/props/PropertyParsingInfo.java

```diff
@@ -839,10 +842,20 @@ private static void parsePropertyValueFile(
                                     + " in "
                                     + indexUnicodeProperties.ucdVersion);
                 }
+                // Unihan 4.0 and earlier implemented multivalued properties by repeating the
+                // property value record instead of using a delimiter.
+                var merger =
+                        propInfo.property.getShortName().startsWith("cjk")
+                                        && indexUnicodeProperties.ucdVersion.compareTo(
+                                                        VersionInfo.UNICODE_4_0)
```

Comment:
nit: testing `... .ucdVersion.getMajor() <= 4` seems more readable

Reply:
More readable, less correct, since it would include Unicode 4.0.1 and 4.1.0.